ansible/roles/pg_standby/templates/recovery.conf.j2

```diff
@@ -0,0 +1,153 @@
+# The command *will* be asked for log files that are not present in the
+# archive; it must return nonzero when so asked.
+#
+# NOTE that the basename of %p will be different from %f; do not
+# expect them to be interchangeable.
+#
+restore_command = 'cp {{ archive_dir }}/%f %p'		# e.g. 'cp /mnt/server/archivedir/%f %p'
+#
+#
+# archive_cleanup_command
+#
+# specifies an optional shell command to execute at every restartpoint.
+# This can be useful for cleaning up the archive of a standby server.
+#
+archive_cleanup_command = 'pg_archivecleanup {{ archive_dir }} %r'
+#
+# recovery_end_command
+#
+# specifies an optional shell command to execute at completion of recovery.
+# This can be useful for cleaning up after the restore_command.
+#
+#recovery_end_command = ''
+#
+#---------------------------------------------------------------------------
+# RECOVERY TARGET PARAMETERS
+#---------------------------------------------------------------------------
+#
+# By default, recovery will rollforward to the end of the WAL log.
+# If you want to stop rollforward at a specific point, you
+# must set a recovery target.
+#
+# You may set a recovery target either by transactionId, by name,
+# or by timestamp. Recovery may either include or exclude the
+# transaction(s) with the recovery target value (ie, stop either
+# just after or just before the given target, respectively).
+#
+#
+#recovery_target_name = ''	# e.g. 'daily backup 2011-01-26'
+#
+#recovery_target_time = ''	# e.g. '2004-07-14 22:39:00 EST'
+#
+#recovery_target_xid = ''
+#
+#recovery_target_inclusive = true
+#
+#
+# Alternatively, you can request stopping as soon as a consistent state
+# is reached, by uncommenting this option.
+#
+#recovery_target = 'immediate'
+#
+#
+# If you want to recover into a timeline other than the "main line" shown in
+# pg_control, specify the timeline number here, or write 'latest' to get
+# the latest branch for which there's a history file.
+#
+#recovery_target_timeline = 'latest'
+#
+#
+# If pause_at_recovery_target is enabled, recovery will pause when
+# the recovery target is reached. The pause state will continue until
+# pg_xlog_replay_resume() is called. This setting has no effect if
+# hot standby is not enabled, or if no recovery target is set.
+#
+#pause_at_recovery_target = true
+#
+#---------------------------------------------------------------------------
+# STANDBY SERVER PARAMETERS
+#---------------------------------------------------------------------------
+#
+# standby_mode
+#
+# When standby_mode is enabled, the PostgreSQL server will work as a
+# standby. It will continuously wait for the additional XLOG records, using
+# restore_command and/or primary_conninfo.
+#
+standby_mode = on
+#
+# primary_conninfo
+#
+# If set, the PostgreSQL server will try to connect to the primary using this
+# connection string and receive XLOG records continuously.
+#
+#primary_conninfo = ''		# e.g. 'host=localhost port=5432'
```

Comment:
This is only needed for streaming replication. Its currently setup for simple log shipping, so the standby is just reading the log files from the archive directory and doesn't need to connect to the main db server. Managing that connection seemed like an extra bit of complexity that was unnecessary to get a simple standby up and running.